app/services/status_handler.rb

```diff
@@ -40,24 +40,24 @@ def mark_moab_not_found
   # @caller_validates_checksums [Boolean] defaults to false.  was this called by code that re-computes checksums to confirm that they match the
   #   values listed in the manifests?
   # @return [void]
-  def set_status_as_seen_on_disk(found_expected_version:, moab_on_storage_validator:, caller_validates_checksums: false)
+  def validate_moab_on_storage_and_set_status(found_expected_version:, moab_on_storage_validator:, caller_validates_checksums: false)
```

Comment:
Took me a long time to realize that validation was happening in catalog to moab and this is where it was happening.